datafusion/physical-plan/src/repartition/mod.rs

```diff
@@ -1333,23 +1335,58 @@ mod tests {
 
         let exec = RepartitionExec::try_new(Arc::new(input), partitioning).unwrap();
 
-        let expected = vec![
-            "+------------------+",
-            "| my_awesome_field |",
-            "+------------------+",
-            "| foo              |",
-            "| bar              |",
-            "| frob             |",
-            "| baz              |",
-            "+------------------+",
-        ];
-
-        assert_batches_sorted_eq!(&expected, &expected_batches);
+        // The results are different from what is expected in snapshots thus i am not using
```

Comment:
I do't understand this comment. batches to string has the same result, so it seems that this is ok.

I think the difference as @blaginin  pointed out on a previous PR is that the batches_to_sort_string sorts the otuput that appears in the test whereas `assert_batches_sorted_eq` sorts before comparison

So TLDR is I think you can just used `bathes_to_sort_string` and update the output